.git-blame-ignore-revs

```diff
@@ -3,3 +3,5 @@
 # add ocamlformat config `wrap-fun-args=false`
 
 75504946eaa6f817550b649df508d61dde12bbda
+# Upgrade to OCamlformat 0.26.0
+a94c9d608ba4f92a0dce03187e817182084158e0
```

Comment:
Is this a welcome addition ? If yes, this PR must not be squashed when ready. (it's not ready yet)